packages/InteractiveProfilingTool-Core.package/IPTTreeModel.class/instance/getIcon..st

```diff
@@ -0,0 +1,6 @@
+model access
+getIcon: aMessageTally
+	self highlightingThreshold <= (self getRelativeTime: aMessageTally) 
```

Comment:
The conditional can be moved to a method called `shouldBeHighlighted` for improved readability

Reply:
Also please add a new line after the message signature